instrumentation/spring/spring-jms/spring-jms-2.0/testing/src/main/java/io/opentelemetry/testing/AbstractJmsTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.testing;
```

Comment:
@laurit thank you for comments.
Is there any convention about naming in `testing` module?

Reply:
I'd use `io.opentelemetry.instrumentation.spring.jms.v2_0`